src/lib/loader/ComponentIo.coffee

```diff
@@ -138,7 +138,7 @@ exports.getSource = (loader, name, callback) ->
     nameParts[0] = ''
 
   if loader.isGraph component
-    nofloGraph.loadFile component, (err, graph) ->
+    fbpGraph.loadFile component, (err, graph) ->
```

Comment:
Same here, isn't. Might be best to make the imports be `fbpGraph = require('fbp-graph').graph` ?